concepts/05 UI Components/DataGrid/07 Choose a Data Management Component/10 TreeList.md

```diff
@@ -0,0 +1,11 @@
+Displays hierarchical data in a tree with grid-like features. Suitable for parent-child structures. 
+
+<img src="/images/treelist/tree-list-choose.png" alt="DevExtreme TreeList" />
+
+Use it for:
+
+- Organizational charts  
```

Comment:
Let's include target key words 

Organizational structures: Customer Relationship Management (CRM) and Human Resource Management (HRM)